File: app/src/main/java/org/zeroxlab/imgorg/lib/Media.java
// vim: et sw=4 sts=4 tabstop=4
package org.zeroxlab.imgorg.lib;

import java.util.Date;

public class Media {
    final public String data;
    final public Date date;

    public Media(String data) {
        this(data, -1);
    }

    Media(String data, long dateInt) {
        this.data = data;
        this.date = new Date(dateInt);
    }
}

File: app/src/main/java/cc/jchu/imgorg/lib/Media.kt
// vim: et sw=4 sts=4 tabstop=4
package cc.jchu.imgorg.lib

import java.util.Date

class Media(
    @JvmField
    val data: String?,
    val dateInt: Long = -1
) {
    @JvmField
    val date: Date

    constructor(data: String?) : this(data, -1) {}

    init {
        date = Date(dateInt)
    }
}
